apps/server/src/utils/time.ts

```diff
@@ -41,16 +41,24 @@ const parse = (valueAsString: string): number => {
  * @param {boolean} fillLeft - autofill left = hours / right = seconds
  * @returns {number} - time string in millis
  */
-export const forgivingStringToMillis = (value: string, fillLeft = true): number => {
+export const forgivingStringToMillis = (value: string, fillLeft: boolean = true): number => {
   let millis = 0;
 
+  //force lower case before AM/PM check
+  value = value.toLowerCase();
```

Comment:
we shouldnt be mutating the parameters in a function. That may lead us to hard to find bugs were the data is changing in unpredictable ways, we can just make a new variable 
`const lowerValue = value.toLowerCase();`